x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/endpoint_details_tabs.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+import styled from 'styled-components';
+import { EuiTabbedContent, EuiTabbedContentTab } from '@elastic/eui';
+
+export enum EndpointDetailsTabsTypes {
```

Comment:
Will this new tab have a url param to reach it? (`show=activity-log`)

if so, maybe sync this up with the url params for consistency

Reply:
Not yet. I'll do so.

Reply:
done 3859af91e0316fa2e57b204964e3dccb657e4e52